Cargo.toml

```diff
@@ -33,6 +33,7 @@ futures = "0.1.14"
 gfx = "0.16"
 num_cpus = "1.2"
 genmesh = "0.4"
+image = "0.15"
```

Comment:
Recording CPU cycles with `perf` on the [PBR example of three](https://github.com/kvark/three-rs/blob/master/examples/pbr.rs) suggests that the major bottleneck is image decoding which is achieved using the `image` crate. Most notably, PNG decoding and converting between RGB / RGBA (etc.) is markedly slow in debug mode - about 10 seconds on my machine for the PBR example. In release mode however the load time is negligible.

The runtime performance and compile time impacts on iteration time, so I try to avoid the `image` crate where feasible. Not all parts are slow, for example `save_buffer` is quick and effective in debug mode. I haven't looked into ways of improving the crate - it's much less effort for me to use `stb_image` + `image_buffer` instead.